www/source/partials/docs/_glossary-launcher.html.md.erb

```diff
@@ -0,0 +1,23 @@
+# <a name="glossary-launcher" id="glossary-launcher" data-magellan-target="glossary-launcher">Launcher</a>
+
+Habitat's Launcher is a sidecar process for the Supervisor which provides a mechanism for launching processes on behalf of the Supervisor. It is the entry point for running the Supervisor and is the Supervisor for the Supervisor. Whereas the Supervisor is able to automatically update itself, the Launcher is currently released a bit differently, by design; it should be rare that the Launcher ever needs to change.
+
+To update your Launchers, you will need to run
+
+```
+hab pkg install core/hab-launcher
+```
+
+on each machine that has a Supervisor, and then restart your Supervisor. This will, by necessity, require a restart of supervised services, so factor that into your planning.
+
+The Launcher is designed to run as process 1; it's extremely minimal by design. It's responsibilities are to:
+
+* Launch the Habitat Supervisor
+* Restart the Habitat Supervisor on shutdown (Supervisor for the Supervisor)
+* Start, Stop, and Restart processes on behalf of the Supervisor
+
+Launcher is specifically designed to enable the Supervisor to self update without shutting down or re-parenting the services that the Supervisor is supervising. It is versioned separately from the Supervisor and should be updated very infrequently since an update of the Launcher could require a system restart if it is running as process 1.
+
+The Supervisor is *always* started by the Launcher. The Launcher will always attempt to start the latest version of the Supervisor package installed on the host. When the Launcher starts the Supervisor a handshake is made over a platform specific [IPC](https://en.wikipedia.org/wiki/Inter-process_communication) channel. The Supervisor communicates with the Launcher with a binary protocol through the Launcher client.
+
+When the Supervisor needs to spawn a long lived process it sends a `Spawn` message to the Launcher and receives a message back with the PID of the running process or a failure message. The Supervisor will watch this process and ask the Launcher to stop or restart it as needed. If the Supervisor crashes or is restarted due to an automatic update it will reattach to all running services and continue operation normally.
```

Comment:
```suggestion
When the Supervisor needs to load and execute a new child process, it sends a `spawn` message to the Launcher. The Launcher returns the process identification (PID) of the running process to the Supervisor if the child process launches successfully, and it returns a failure message if it does not. The Supervisor watches the spawned child process and asks the Launcher to stop or restart it, as needed. If the Supervisor crashes or is restarted by an update, it then re-attaches itself to all running services and continues normal operation.
```
Let me know if I have missed your meaning.

Reply:
Beautiful. Thanks for cleaning up that last paragraph, you can tell I was getting tired towards the end there.